refinery/units/crypto/hash/password_hashes.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Implements password hashing algorithms.
+"""
+import hashlib
+import binascii
+
+from refinery.units.crypto.hash import HashUnit
+
+
+class ntlm(HashUnit):
+    """
+    Returns the Windows NTLM hash of the input.
+    """
+    def _algorithm(self, data: bytes) -> bytes:
+        digest = hashlib.new('md4', data.decode("utf-8").encode("utf-16le")).digest()
+        return binascii.hexlify(digest)
```

Comment:
```suggestion
        return hashlib.new('md4', data.decode(self.codec).encode("utf-16le")).digest()
```

Reply:
We should remove everything `binascii` and return the raw hash bytes from `_algorithm`; this is what the `HashUnit` expects, and you can get the plaintext variant of the hash by running `ntlm -t`. With `binascii`, the `ntml` unit would still have a command line flag called `-t` which claims to convert the hash to hexadecimal representation, but it would instead give you the hexadecimal encoding of the hexadecimal encoding of the hash.

Reply:
The class variable `self.codec` is currently hard-coded to UTF8, but even in the future it will always represent the encoding used by refinery units for text data.

Reply:
Done, tested & squashed, thanks!